module_utils/network_lsr/ethtool.py

```diff
@@ -46,7 +50,7 @@ def get_perm_addr(ifname):
             res = ecmd.tobytes()
         except AttributeError:  # tobytes() is not available in python2
             res = ecmd.tostring()
-        _, size, perm_addr = struct.unpack("II%is" % MAX_ADDR_LEN, res)
+        dummy, size, perm_addr = struct.unpack("II%is" % MAX_ADDR_LEN, res)
```

Comment:
The inclusive open source community orientation training at

https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/


recommends in

https://trainingportal.linuxfoundation.org/learn/course/inclusive-open-source-community-orientation-lfc102/making-change/making-change

to avoid the term "Dummy Value" and use "Placeholder Value" instead. I do not see an explanation for this.

Reply:
I guess I have to register to see the contents of that link?  At any rate - what is the suggestion?
```python
         placeholder, size, perm_addr = struct.unpack("II%is" % MAX_ADDR_LEN, res)
```
?

Reply:
why do you want to replace `_` in the first place? I though that is pretty common and pythonic.

An alternative name might be `unused`. While I am surprised to hear "dummy" is discouraged, `unused` anyway seems a better name.

Reply:
> why do you want to replace `_` in the first place? I though that is pretty common and pythonic.

https://github.com/linux-system-roles/network/pull/362#discussion_r589668462
tl;dr - `_` is also used for i18n, and Ansible wants to reserve `_` for future use.

> 
> An alternative name might be `unused`. While I am surprised to hear "dummy" is discouraged, `unused` anyway seems a better name.

"dummy" is a derogatory word which can be seen as non-inclusive

`unused` works for me

Reply:
I'll note that "dummy" is used in many places in the network role:
```
> git grep -c dummy
examples/dummy_simple.yml:4
library/network_connections.py:1
module_utils/network_lsr/argument_validator.py:1
pylintrc:2
tests/ensure_provider_tests.py:1
tests/playbooks/tests_dummy.yml:5
tests/tasks/create_dummy_profile.yml:1
tests/tasks/manage_test_interface.yml:9
tests/tests_dummy_nm.yml:2
tests/tests_helpers_and_asserts.yml:2
```

Reply:
> I guess I have to register to see the contents of that link? At any rate - what is the suggestion?

You can easily login with your Red Hat Google account if you like. The course was recommended in the past and you can get a Fedora badge if you complete it.

> I'll note that "dummy" is used in many places in the network role:

It is also an official network interface type, see https://developers.redhat.com/blog/2018/10/22/introduction-to-linux-interfaces-for-virtual-networking/#dummy so we cannot get it completely removed.

> 
> ```
> > git grep -c dummy
> examples/dummy_simple.yml:4
> library/network_connections.py:1
> module_utils/network_lsr/argument_validator.py:1
> pylintrc:2

This is a good find since it shwos that `unused` is configured as an expectedly not used variable name, so we can use that.

> tests/ensure_provider_tests.py:1
> tests/playbooks/tests_dummy.yml:5
> tests/tasks/create_dummy_profile.yml:1
> tests/tasks/manage_test_interface.yml:9
> tests/tests_dummy_nm.yml:2
> tests/tests_helpers_and_asserts.yml:2
> ```